### DISCUSSION TITLE
Editsign Copying

### DISCUSSION THREAD

**Discussion by Pheotis** [EssentialsX/Essentials#3973] [Ideas and Feature Suggestions] [answered]
#### Feature description
Per #3467, it is currently possible to copy text that is already on a sign using `/editsign set n` via tab complete.
This indirectly adds the capacity to copy a sign, albeit not in a streamlined manner.

A dedicated system to duplicate signs would make this process much easier.
For example, `/editsign copy` and `/editsign paste`.

#### How the feature is useful
If you created an oak sign, but want to migrate the text to a dark oak sign, it currently takes eight commands (including tab completes).
A singular command would make that process much easier.

It is also a good solution for allowing players to duplicate signs without needing to give them the full `minecraft.nbt.copy` node.

**Comment by JRoy** [comment]
See #3989

**Comment by JRoy** [comment, marked as answer]
Hello, this feature has been added in the latest dev build! Download the latest dev build from [here](https://essentialsx.net/downloads.html)!
